activesupport/test/core_ext/duration_test.rb

```diff
@@ -224,6 +224,9 @@ def test_comparable
     assert_equal(1, (61 <=> 1.minute))
   end
 
+  # ISO8601 string examples are taken from ISO8601 gem at https://github.com/arnau/ISO8601/blob/b93d466840/spec/iso8601/duration_spec.rb
+  # published under the conditions of MIT license: https://github.com/arnau/ISO8601/blob/b93d466840/LICENSE
+
```

Comment:
The license is pretty clear about what we need to include here:
```
The above copyright notice and this permission notice shall be
included in all copies or substantial portions of the Software.
```
So we need to include the license text in a comment here.